config/api_client.go

```diff
@@ -0,0 +1,55 @@
+package config
+
+import (
+	"fmt"
+	"net/url"
+
+	"github.com/CircleCI-Public/circleci-cli/api/graphql"
+	"github.com/CircleCI-Public/circleci-cli/api/rest"
+	"github.com/CircleCI-Public/circleci-cli/settings"
+)
+
+var (
+	compilePath = "compile-config-with-defaults"
+)
```

Comment:
This one doesn't seem to change. Can we make this a `const`?

```suggestion
const compilePath = "compile-config-with-defaults"
```